src/overlays/actors/ovl_En_Invadepoh/z_en_invadepoh.c

```diff
@@ -20,492 +251,4503 @@ const ActorInit En_Invadepoh_InitVars = {
+    if (0.9999f <= this->clockTime) {
+        this->unk38A = 1;
+    }
+}
+
+void func_80B47830(EnInvadepoh* this) {
+    this->collider.base.atFlags &= ~1;
+    this->collider.base.acFlags &= ~1;
+    this->collider.base.ocFlags1 |= 1;
+    SkelAnime_ChangeAnimDefaultRepeat(&this->skelAnime, &D_060006C8);
+    func_800BCB70(&this->actor, 0x4000, 0xFF, 0, 0x10);
+    this->alienAlpha = 0xFF;
+    this->actor.draw = func_80B4DB14;
+    this->drawAlien = true;
+    this->drawAlienDeathEffect = false;
+    this->alienBeamAlpha = 0;
+    this->actionTimer = 8;
+    this->counter = 0;
+    this->actor.flags |= 0x80000000;
+    this->actionFunc = func_80B478F4;
+}
+
+void func_80B478F4(EnInvadepoh* this, GlobalContext* globalCtx) {
+    func_80B44A90(this, globalCtx);
+    this->actionTimer--;
+    if (this->actionTimer <= 0) {
+        func_80B47938(this);
+    }
+}
+
+void func_80B47938(EnInvadepoh* this) {
+    this->collider.base.atFlags &= ~1;
+    this->collider.base.acFlags &= ~1;
+    this->collider.base.ocFlags1 &= ~1;
+    SkelAnime_ChangeAnimDefaultRepeat(&this->skelAnime, &D_06000608);
+    this->actor.flags &= ~1;
+    this->actionTimer = 0xA;
+    this->alienAlpha = 0xFF;
+    this->actor.draw = func_80B4DB14;
+    this->drawAlien = true;
+    this->drawAlienDeathEffect = false;
+    this->alienBeamAlpha = 0xFF;
+    this->actor.flags |= 0x80000000;
+    this->actionFunc = func_80B479E8;
+}
+
+void func_80B479E8(EnInvadepoh* this, GlobalContext* globalCtx) {
+    s16 unk2F2;
+    u32 index;
+
+    func_80B44B78(this);
+    if (this->counter < 5) {
+        if (D_80B4ED30[this->counter].x > 0.0f) {
+            this->drawAlien = true;
+            this->alienBeamAlpha = 0xFF;
+            Math_Vec3f_Copy(&this->actor.scale, &D_80B4ED30[this->counter]);
+        } else {
+            this->drawAlien = false;
+            this->alienBeamAlpha = 0;
+        }
+
+    } else {
+        this->drawAlien = false;
+        this->alienBeamAlpha = 0;
+    }
+
+    unk2F2 = this->counter;
+    if ((unk2F2 >= 2) && (unk2F2 < 9)) {
+        index = unk2F2 - 2;
+        if (D_80B4ED6C[index].x > 0.0f) {
+            this->drawAlienDeathEffect = true;
+            Math_Vec3f_Copy(&this->alienDeathEffectScale, &D_80B4ED6C[index]);
+        } else {
+            this->drawAlienDeathEffect = false;
+        }
+
+    } else {
+        this->drawAlienDeathEffect = false;
+    }
+
+    this->counter++;
+    if (this->actionTimer == 8) {
+        Actor_Spawn(&globalCtx->actorCtx, globalCtx, ACTOR_EN_CLEAR_TAG, this->actor.world.pos.x,
+                    this->actor.world.pos.y - 10.0f, this->actor.world.pos.z, 0, 0, 3, 200);
+    }
+
+    if (this->actionTimer == 8) {
+        func_800BBA88(globalCtx, &this->actor);
+    }
+
+    this->actionTimer--;
+    if (this->actionTimer <= 0) {
+        func_80B43AF0(this->actor.params & 7);
+        func_80B43AB0();
+        Item_DropCollectible(globalCtx, &this->actor.world.pos, 8);
```

Comment:
```suggestion
        Item_DropCollectible(globalCtx, &this->actor.world.pos, ITEM00_ARROWS_30);
```
That makes sense right?

Reply:
The aliens drop those when you kill them so I think it makes sense